### PR TITLE
zh-cn: revise the translation of "splash page"

### PR DESCRIPTION
**Pull request by hanyujie2002**
relates to #18313

### CODE REVIEW

File: files/zh-cn/learn/html/multimedia_and_embedding/mozilla_splash_page/index.md [hunk 1]
```diff
@@ -1,94 +1,97 @@
...
 
-访问 [originals](https://github.com/roy-tian/learning-area/tree/master/html/multimedia-and-embedding/mdn-splash-page-start/originals) 文件夹中不同的图片文件，然后用同样的方法将它们下载到本地。现在，你需要将这些图片文件保存到不同的目录下，因为在这些图片准备好被使用之前，你需要使用图像编辑器来处理这些图片。
+以同样的方式访问 [originals](https://github.com/mdn/learning-area/tree/main/html/multimedia-and-embedding/mdn-splash-page-start/originals) 目录中的多个图片，并将它们保存下来；你需要将它们保存在不同的目录中，因为你需要在使用之前使用图形编辑器对其中一些图片进行处理。
 
-> **备注：** 这个示例的 HTML 文件包含一些页面的 CSS 样式。你不需要去碰 CSS 的内容，而只是 {{htmlelement("body")}} 元素中的 HTML 部分，只要你插入了正确的标记，样式就会正确显示。
+或者，你可以使用在线编辑器，如 [CodePen](https://codepen.io/)、[JSFiddle](https://jsfiddle.net/) 或 [Glitch](https://glitch.com/)。
+
+> **备注：** 示例 HTML 文件包含大量 CSS，用于为页面设置样式。你不需要修改 CSS，只需修改 {{htmlelement("body")}} 元素内的 HTML 即可。只要插入正确的标记，样式就会使其显示正确。
+>
+> 如果你遇到困难，可以在我们的[交流渠道](/zh-CN/docs/MDN/Community/Communication_channels)中联系我们。
```
Comment (yin1999) [inline files/zh-cn/learn/html/multimedia_and_embedding/mozilla_splash_page/index.md:39]:
```suggestion
> 如果你遇到困难，可以在我们的[沟通渠道](/zh-CN/docs/MDN/Community/Communication_channels)中联系我们。
```

File: files/zh-cn/learn/html/multimedia_and_embedding/mozilla_splash_page/index.md [hunk 2]
```diff
@@ -1,94 +1,97 @@
...
 
 - `firefox_logo-only_RGB.png`
 - `firefox-addons.jpg`
 - `mozilla-dinosaur-head.png`
 
-给它们取个有意义的名字，例如`firefoxlogo400.png` 和`firefoxlogo120.png`。
+给它们取些有意义的名字，例如 `firefoxlogo400.png` 和 `firefoxlogo120.png`。
 
-这些图片将和 `mdn.svg` 一起作为`further-info`区内资源链接的图标和网站页眉的火狐图标。将这些图片副本保存在与 `index.html`文件的同一个目录下。
+这些图片将与 `mdn.svg` 一起，作为链接到其他资源的图标，放在 `further-info` 区域内。你还需要在站点头部中链接到 Firefox 标志。你应将所有这些图片的副本保存在与 `index.html` 相同的目录中。
```
Comment (yin1999) [inline files/zh-cn/learn/html/multimedia_and_embedding/mozilla_splash_page/index.md:55]:
```suggestion
这些图片将与 `mdn.svg` 一起，作为链接到其他资源的图标，放在 `further-info` 区域内。你还需要在站点头部中链接 Firefox 标志。你应将所有这些图片的副本保存在与 `index.html` 相同的目录中。
```

**Comment by hanyujie2002** [comment]
Thank you 👍👍👍 @PassionPenguin

**Review by yin1999 — APPROVED** [review]
Thanks @hanyujie2002

File: files/zh-cn/learn/html/multimedia_and_embedding/mozilla_splash_page/index.md [hunk 3]
```diff
@@ -1,94 +1,97 @@
...
-        模块的其他文章。
+        在开始这个测验之前，你应该已经学过本模块的其他文章。
       </td>
     </tr>
     <tr>
       <th scope="row">目标：</th>
       <td>
-        测试这些知识的掌握程度：在页面中嵌入图片和视频，框架，和 HTML
-        响应式图片技术。
+        测试对这些知识的掌握程度：在页面中嵌入图片和视频、框架、以及 HTML 响应式图片技术。
```
Comment (yin1999) [inline files/zh-cn/learn/html/multimedia_and_embedding/mozilla_splash_page/index.md:23]:
```suggestion
        测试对这些知识的掌握程度：在页面中嵌入图片和视频、框架，以及 HTML 响应式图片技术。
```

**Pull request merged**